packages/airnode-abi/jest.config.js

```diff
@@ -2,5 +2,4 @@ const config = require('../../jest.config.base');
 
 module.exports = {
   ...config,
-  // Add custom settings below
```

Comment:
I've removed these comments as I didn't see much added value and I was modifying those configuration files.